sentence_transformers/evaluation/RerankingEvaluator.py

```diff
@@ -194,7 +197,10 @@ def __call__(
         self.store_metrics_in_model_card_data(model, metrics, epoch, steps)
         return metrics
 
-    def compute_metrices(self, model):
+    def compute_metrices(
+        self,
+        model: SentenceTransformer,
```

Comment:
There are a few places where you can remove the comma on the last item in the argument list, and then `ruff`/`pre-commit run` will put everything on one line.